kernel/prepare/pkg.yaml

```diff
@@ -42,9 +42,6 @@ steps:
         make mrproper
       - |
         cd /toolchain && git clone https://github.com/a13xp0p0v/kconfig-hardened-check.git
-      - |
-        patch -p1 < /pkg/patches/hardening.kconfig.patch
```

Comment:
no more hardening patches needed?

Reply:
this was called confusingly, but it patched Kconfig to make hardening checks pass :-D so it's actually better now than it was